### PR TITLE
Update an FAQ according to discussion

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Update an FAQ added in https://github.com/pingcap/docs-cn/pull/8997 according to discussion.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Chad20N13 — APPROVED** [review]
LGTM

**Review by yiwen92 — APPROVED** [review]
Disk_Size` 表示压缩后表的大小，根据 `Approximate_Size` 和 `store_size_amplification` 计算出。 ——强调一下是估算吧

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -230,9 +236,12 @@ GROUP BY db_name , table_name;
...
 - `@dbname`：数据库名称。
 - `@table_name`：目标表的名称。
-- `@size_amplification`：集群压缩比的平均值。要获取该信息，查看 Grafana 监控 **PD - statistics balance** 面板下各节点的 `Size amplification` 指标，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
 
-此外，以上语句中 `Approximate_Size` 表示压缩前表的大小，`Disk_Size` 表示压缩后表的大小。
+此外，以上语句中：
+
+- `store_size_amplification` 表示集群压缩比的平均值。除了使用 `SELECT * FROM METRICS_SCHEMA.store_size_amplification;` 语句进行查询以外，你还可以查看 Grafana 监控 **PD - statistics balance** 面板下各节点的 `Size amplification` 指标来获取该信息，集群压缩比的平均值即为所有节点的 `Size amplification` 平均值。
+- `Approximate_Size` 表示压缩前表的单副本大小，该值为估算值，并非准确值。
+- `Disk_Size` 表示压缩后表的大小，根据 `Approximate_Size` 和 `store_size_amplification` 计算出。
```
Comment (TomShawn) [inline faq/manage-cluster-faq.md:244]:
```suggestion
- `Disk_Size` 表示压缩后表的大小，可根据 `Approximate_Size` 和 `store_size_amplification` 得出估算值。
```

Comment (TomShawn) [reply faq/manage-cluster-faq.md]: @yiwen92 FYI

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**